pkg/fileutil/fileutil_test.go

```diff
@@ -34,8 +35,14 @@ func TestIsDirWriteable(t *testing.T) {
 	if err := os.Chmod(tmpdir, 0444); err != nil {
 		t.Fatalf("unexpected os.Chmod error: %v", err)
 	}
-	if err := IsDirWriteable(tmpdir); err == nil {
-		t.Fatalf("expected IsDirWriteable to error")
+	me, err := user.Current()
```

Comment:
err can be non-nil when cross-compiled.
So I would like to keep current behavior.
https://www.google.com/search?q=user%3A+Current+not+implemented